workscore.go

```diff
@@ -35,32 +35,32 @@ func (w WorkScore) Multiply(in int) (WorkScore, error) {
 
 type WorkScoreStructure struct {
 	// DataKilobyte accounts for the network traffic per kilobyte.
-	DataKilobyte WorkScore `serix:"0,mapKey=dataKilobyte"`
+	DataKilobyte WorkScore `serix:"0,mapKey=workScoreDataKilobyte"`
```

Comment:
```suggestion
	DataKilobyte WorkScore `serix:"0,mapKey=workScoreDataKibibyte"`
```

Let make this `Kibibyte`, actually.